src/_help/customization-options/operations-navigation.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Operations and navigation
+---
+
+- TOC
+{:toc}
+
+Bump.sh offers several methods for sorting your API operations to provide a smooth and coherent reading structure, based on either the tags or the endpoints of your API definition.
```

Comment:
```suggestion
Bump.sh offers several methods for sorting your API operations to provide a smooth and coherent reading structure, based on either the tags or the operation names and URLs.
```

WDYT of this change? It seems a bit more clear to me here.

Reply:
Way much better. I suppose that I was just proud to correctly use "API definition" :D

Reply:
Haha, and it was perfectly used ;)